source/common/access_log/access_log_impl.h

```diff
@@ -121,7 +121,7 @@ class OrFilter : public OperatorFilter {
  */
 class NotHealthCheckFilter : public Filter {
 public:
-  NotHealthCheckFilter() {}
```

Comment:
I think (most case of this lines) can be just removed, no?

Reply:
I _believe_ you're right. I can go through pulling public default special members and verify build/tests

Reply:
Sure, I think this can be a separate PR though, best to separate that from the noise. I'll merge this one.